samples/ingestion/ingestion-client/Setup/guide.md

```diff
@@ -55,7 +55,7 @@ To test, we recommend you use [Microsoft Azure Storage Explorer](https://azure.m
 
 To deploy the required infrastructure, click the button below:
```

Comment:
There's still a reference to the Setup folder in Readme.md:


| `Setup` | ARM template to setup the project on Azure. |

Reply:
Updated